taqueria-plugin-ligo/compile.ts

```diff
@@ -28,7 +28,7 @@ const getCompileCommand = (opts: Opts) =>
 
 		const inputFile = getInputFilename(opts)(sourceFile);
 		const baseCommand =
-			`DOCKER_DEFAULT_PLATFORM=linux/amd64 docker run --rm -v \"${projectDir}\":/project -w /project ligolang/ligo:next compile contract ${inputFile}`;
+			`DOCKER_DEFAULT_PLATFORM=linux/amd64 docker run --rm -v \"${projectDir}\":/project:z -w /project ligolang/ligo:next compile contract ${inputFile}`;
```

Comment:
How does adding `:z` fix the problem?

Reply:
It does not. Sorry, I have marked the PR as draft.
The `:z` is useful anyway, for SELinux scenarios, I just wanted to push, to see what happens to all tests as a result of this change.